core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -32,6 +32,7 @@
  */
 public abstract class RetryingClient<I extends Request, O extends Response>
         extends SimpleDecoratingClient<I, O> {
+    private static final int DEFAULT_MAX_ATTEMPTS = 5;
```

Comment:
That was my first thought that the parameter would be set up from system property. However, it  is not really related to system so I hard-coded it with 5.  I still can't find it, do you have any idea?